Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -201,16 +193,7 @@ define([
          */
         this.enablePickFeatures = defaultValue(options.enablePickFeatures, true);
```

Comment:
This should just be `true` here, since `options` may be a promise and not have `enablePickFeatures` property.  Then in `reinitialize` we should do `this.enablePickFeatures = defaultValue(options.enablePickFeatures, this.enablePickFeatures);` so that it keeps whatever the setting was if it's omitted from the reinitialize options.